app/src/main/java/com/example/parkingsystemjava/DialogFragmentParkingLots.java

```diff
@@ -0,0 +1,48 @@
+package com.example.parkingsystemjava;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import androidx.fragment.app.DialogFragment;
+import com.example.parkingsystemjava.databinding.DialogFragmentParkingLotsBinding;
+import com.example.parkingsystemjava.mvp.contract.DialogFragmentContract;
+import com.example.parkingsystemjava.mvp.listener.CustomListener;
+import com.example.parkingsystemjava.mvp.presenter.DialogFragmentPresenter;
+import com.example.parkingsystemjava.mvp.view.DialogFragmentView;
+
+public class DialogFragmentParkingLots extends DialogFragment {
+    private DialogFragmentParkingLotsBinding binding;
+    private DialogFragmentContract.PresenterContract presenter;
+    private CustomListener customListenerDialogFragment;
+
+    public DialogFragmentParkingLots() {
+    }
+
+    public static DialogFragmentParkingLots newInstance() {
+        DialogFragmentParkingLots fragment = new DialogFragmentParkingLots();
+        Bundle args = new Bundle();
```

Comment:
this is not necessary if you are not passing anything between activity and fragment

Reply:
done,  deleted